tests/antares/services/api_services/test_study_api.py

```diff
@@ -199,3 +199,103 @@ def test_create_binding_constraint_fails(self):
                 match=f"Could not create the binding constraint {constraint_name}: {self.antares_web_description_msg}",
             ):
                 self.study.create_binding_constraint(name=constraint_name)
+
+    def test_read_study_api(self):
+        json_study = {
+            "id": "22c52f44-4c2a-407b-862b-490887f93dd8",
+            "name": "test_read_areas",
+            "version": "880",
+        }
+
+        json_ui = {
+            "zone": {
+                "ui": {"x": 0, "y": 0, "color_r": 230, "color_g": 108, "color_b": 44, "layers": "0"},
+                "layerX": {"0": 0},
+                "layerY": {"0": 0},
+                "layerColor": {"0": "230, 108, 44"},
+            }
+        }
+
+        json_thermal = [
```

Comment:
To simplify your test you can just replace the json you introduce by:
```python
json_thermal = []
json_renewable = []
json_st_storage = []
json_properties = {}
```
and your test will still pass.
You can even remove these variables and simply pur the [] or {} inside the .get() to make it even simpler